MEMBER_REPRESENTATION.md

```diff
@@ -0,0 +1,9 @@
+# Member Representation
+
+The following document lists OpenJS Foundation members and their representation to any standards organizations:
+
+* <firstname> <lastname>
```

Comment:
richard and i are delegates to tc39 already, should we be added here now or do separate prs?

Reply:
Let's do separate PRs for adding individuals